src/Codeception/Test/Feature/CodeCoverage.php

```diff
@@ -23,11 +25,9 @@ public function codeCoverageEnd($status, $time)
         if (!$codeCoverage) {
             return;
         }
-        $linesToBeCovered = \PHPUnit_Util_Test::getLinesToBeCovered(get_class($this->testClassInstance), 'test');
-        $linesToBeUsed = \PHPUnit_Util_Test::getLinesToBeUsed(get_class($this->testClassInstance), 'test');
 
         try {
-            $codeCoverage->stop(true, $linesToBeCovered, $linesToBeUsed);
```

Comment:
Those values are taken from annotations like `@covers`. I don't think they can be implemented for multiple formats, and they were not documented to be used inside Cept or Cest files. So for now they are just ignored. Probably in future we can come to idea how to get it to use